data/org.containers.hirte.Node.xml

```diff
@@ -58,7 +58,9 @@
     <method name="ListUnits">
       <arg name="units" type="a(ssssssouso)" direction="out" />
     </method>
-    <method name="Reload" />
+    <method name="SelNodeLogLevel">
```

Comment:
```suggestion
    <method name="SetNodeLogLevel">
```

Reply:
Since it is on the Node API, simply `SetLogLevel` is enough :)

Reply:
Agree